spec/models/group_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe Group do
+  required_props = %i(
+    name
+    moderator_email
```

Comment:
Hey @npauzenga, I think I would go the association route. It's completely fine for a model to have a `has_many` relationship with another model and at the same time a `belongs_to` relationship with the same type of model.